resources/styles/molecules/tabs/BpTabs.vue

```diff
@@ -96,117 +100,143 @@ export default {
-                ? (this.wrap ? this.firstTabKey : null)
-                : this.tabKeys[this.currentFocusTabIndex + 1]
-        },
-        prevTabKey () {
-            return this.currentFocusTabIndex === 0
-                ? (this.wrap ? this.lastTabKey : null)
-                : this.tabKeys[this.currentFocusTabIndex - 1]
-        },
-    },
-
-    mounted () {
-        this.setInitFocusTab()
-        this.setInitActiveTab()
-        this.attachKeyboardListeners()
-    },
-
-    methods: {
-        setInitActiveTab () {
-            if (location.hash && this.tabKeys.indexOf(location.hash.substr(1)) !== -1) {
-                this.activeTab = location.hash.substr(1)
-            } else if (this.initialTab) {
-                this.activeTab = this.initialTab
+    const initialTabComp = computed({
+        get: () => {
+            if (!props.initialTab) {
+                var [firstTab] = Object.keys(props.tabs)
+                return firstTab
             } else {
-                [this.activeTab] = this.tabs
+                return props.initialTab
             }
-        },
-        setFocusTab (tab) {
-            this.currentFocusTab = tab
-            this.selectTab(tab)
-        },
-        setInitFocusTab () {
-            this.currentFocusTab = this.initialTab
-        },
-        selectTab (tab) {
-            this.activeTab = tab
-            if (this.setHash) {
-                history.replaceState(undefined, undefined, `#${this.activeTab}`)
-            }
-        },
-        isActive (tab) {
-            return tab === this.activeTab ? this.activeClass : ''
-        },
-        isActiveTab (tab) {
-            return this.activeTab === tab
-        },
-        isFocusedTab (tab) {
-            return this.currentFocusTab === tab
-        },
-        navigateNextTab (evt) {
-            if (this.nextTabKey) {
-                evt.preventDefault();
-                const nextTabButton = this.$el.querySelector(`[id="${this.nextTabKey}"]`)
-                nextTabButton.focus()
-            }
-        },
-        navigatePrevTab (evt) {
-            if (this.prevTabKey) {
-                evt.preventDefault();
-                const prevTabButton = this.$el.querySelector(`[id="${this.prevTabKey}"]`)
-                prevTabButton.focus()
+        }
+    })
+
+    const wrapComp = computed({
+        get: () => {
+            if (!props.wrap) {
+                return !props.vertical
+            } else {
+                return props.wrap
             }
-        },
-        attachKeyboardListeners () {
-            const firstTabEl = this.$el.querySelector(`[id="${this.firstTabKey}"]`)
-            const lastTabEl = this.$el.querySelector(`[id="${this.lastTabKey}"]`)
+        }
+    })
 
-            this.$el.addEventListener('keydown', event => {
-                if (event.keyCode === 36) {
-                    event.preventDefault()
+    const tabKeys = computed({
+        get: () => Object.keys(props.tabs)
```

Comment:
Not explicitly wrong, but I believe another Vue standard is that if you explicitly set a getter for a computer property, you do so only because you are also setting a setter. If you're only using a getter (which is the default for computed properties), you can streamline this by just doing the following:
```
const tabKeys = computed(() => Object.keys(props.tabs))
```